rotkehlchen/chain/evm/decoding/gearbox/balances.py

```diff
@@ -60,8 +64,16 @@ def query_balances(self) -> 'BalancesSheetType':
             )
             for user_address in addresses_with_deposits
         ]
+        chunk_size, call_order = get_chunk_size_call_order(
+            evm_inquirer=self.evm_inquirer,
+            arbiscan_chunksize=ARBITRUM_ETHERSCAN_MULTICALL_CHUNK_SIZE,
+        )
```

Comment:
I'm seeing that we do this call for every protocol. Wouldn't it be better to do it only once and pass it as argument in https://github.com/rotki/rotki/blob/6742a0e43f137d6b7830b8962a28ca758358bb4e/rotkehlchen/chain/aggregator.py#L1142 ?